### ISSUE TITLE
Monitoring - bypass suggestion

### ISSUE THREAD

**Issue by vcvrackideas** [n0jo/rackwindows#8]
Hi Jens! Thank you for implementing this, very useful. Could you considering to add a bypass button in this module? There is of course off value already. But when I switch to it from AURATONE preset I get a huge amount of clicks and this annoys me a bit. Also it could be useful to make off bot headphones and monitors to off. Since v1 doesn't have any bypass option on its modules I think this button would be pretty useful for fast A\B. 
Cheers!

**Comment by n0jo** [comment]
Hi Artem, thanks for checking the module. 

> Could you considering to add a bypass button in this module?

I've discussed this just recently and I'm a bit torn. I agree bypass would be very useful, but on the other hand I don't really want to add a feature that becomes obsolete with v2 in a few months. Once a bypass button exists, I can't easily remove it without potentially breaking someone's patches. Let me think about it. 

Could you please tell me more about the clicking from Aurat and which OS you are on? Obviously it should not do this. Do Vinyl and Phone work without the clicks?  All three of them use the same biquad bandpass, just with different settings.

**Comment by vcvrackideas** [comment]
Yeah no complains on switching between Vinyl, Auratone and Phone. But the off button is on the other side behind pretty clickable SUB, Slew and even Peaks mode that what I meant on bypass button ;-) Cheers!

**Comment by n0jo** [comment]
Oh, you mean getting to Aurat involves a lot of clicking because of the mode switching. I wonder if I could make the mode labels clickable with transparent buttons or something, so there would be no need to turn the knob. Or else possibly crossfade between modes to reduce clicks. Or both, we'll see :-)

**Comment by sumpygump** [comment]
As a workaround, you can right click the knob and enter the value and press enter. For example, the value 7 will switch the knob to Aurat, and then the value 0 will switch the knob to off.
